### PR TITLE
Add ROI and binning handling

### PR DESCRIPTION
**Pull request by Kotochleb**

### CODE REVIEW

File: happypose_ros/happypose_ros/camera_wrapper.py [hunk 1]
```diff
@@ -74,43 +88,24 @@ def __init__(
         # Register callback depending on the configuration
         self._image_approx_time_sync.registerCallback(self._on_image_data_cb)
 
-    def image_guarded(func: Callable[..., RetType]) -> Callable[..., RetType]:
-        """Decorator, checks if image was already received.
+    def data_recieved_guarded(func: Callable[..., RetType]) -> Callable[..., RetType]:
```
Comment (MedericFourmy) [inline happypose_ros/happypose_ros/camera_wrapper.py:91]: "data_received_guarded"

File: happypose_ros/happypose_ros/camera_wrapper.py [hunk 2]
```diff
@@ -5,10 +5,12 @@
 
 from rclpy.node import Node
 from rclpy.time import Time
+from rclpy.qos import qos_profile_sensor_data
```
Comment (MedericFourmy) [inline happypose_ros/happypose_ros/camera_wrapper.py:8]: Add a comment or link explaining what qos_profile_sensor_data is meant to be

File: happypose_ros/happypose_ros/camera_wrapper.py (hunk 1 above)
Comment (MedericFourmy) [reply happypose_ros/happypose_ros/camera_wrapper.py:91]: actually lots of "recieved" in the code

**Comment by Kotochleb** [comment]
Requires https://github.com/agimus-project/happypose_ros/pull/17 to be merged. Afterwards, it will be ready for review

**Review by MedericFourmy — APPROVED** [review]
LGTM

**Pull request merged**